libs/@guardian/source-foundations/src/_deprecated/typography/types.ts

```diff
@@ -0,0 +1,4 @@
+/** @deprecated Type will be removed in a future major release */
```

Comment:
is this message correct? should it be `TypographySizes`?

```suggestion
/** @deprecated TypographySizes will be removed in a future major release */
```